pyvelox/pyvelox.h

```diff
@@ -394,7 +394,8 @@ static void addVectorBindings(
           })
       .def("encoding", &BaseVector::encoding)
       .def("append", [](VectorPtr& u, VectorPtr& v) { appendVectors(u, v); })
-      .def("resize", &BaseVector::resize);
+      .def("resize", &BaseVector::resize)
+      .def("slice", [](VectorPtr& u, vector_size_t offset, vector_size_t length) { return u->slice(offset, length);});
```

Comment:
Currently, this will only support slicing by offset and length. Should we also have pythonic slicing by mentioning `[start, step, end]`.?

Reply:
Yes that will be great to have. 
Can you also add some tests around this ?

Reply:
Added tests now, and for now we aren't supporting step other than 1.